src/transformers/models/align/processing_align.py

```diff
@@ -82,15 +126,28 @@ def __call__(self, text=None, images=None, padding="max_length", max_length=64,
             - **pixel_values** -- Pixel values to be fed to a model. Returned when `images` is not `None`.
         """
         if text is None and images is None:
-            raise ValueError("You have to specify either text or images. Both cannot be none.")
+            raise ValueError("You must specify either text or images.")
 
         if text is not None:
-            encoding = self.tokenizer(
-                text, padding=padding, max_length=max_length, return_tensors=return_tensors, **kwargs
-            )
+            text_kwargs = {
+                **self.processing_kwargs["text_kwargs"],
```

Comment:
So this is basically creating a dictionary with the default, and is updates with the kwargs. 
1. It's very nice
2. It's a tad bit not explicit enough for me (for newbies, it would be hard to guess that this is what's happening IMO)
Thus I am wondering if there is a way to leverage the `ProcessingKwargs` class rather than use the class's keys. 
For that what I mean is something along the line of `self.processing_kwargs.update_with_kwargs("text_kwargs",**kwargs)`  could be more understandable? (I don't know)